packages/ra-ui-materialui/src/button/CreateButton.js

```diff
@@ -40,12 +34,11 @@ const CreateButton = ({
     classes = {},
     translate,
     label = 'ra.action.create',
-    size = 'small',
     ...rest
 }) => (
     <Responsive
         small={
-            <Button
+            <MuiButton
```

Comment:
I don't get this change.Why?

Reply:
Because we're using our own `Button` for the `medium` size, and the names collide.

Reply:
Oh I did not see that we were using the MUI button for both before... :+1: